Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM docker.io/centos:centos8
 
-ARG PKGS_LIST=main-packages-list.txt
+ENV PKGS_LIST=main-packages-list.txt
+ARG EXTRA_PKGS_LIST
 
-COPY ${PKGS_LIST} ${PATCH_LIST} /tmp/
+COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST:-$PKGS_LIST} ${PATCH_LIST} /tmp/
```

Comment:
A blank "PATCH_LIST" has the same problem that blank EXTRA_PKGS_LIST has, can we also default it to PKGS_LIST if unset?

Reply:
yep, done